src/api/cvc4cpp.cpp

```diff
@@ -5594,6 +5594,11 @@ void Solver::resetAssertions(void) const
   CVC4_API_SOLVER_TRY_CATCH_END;
 }
 
+Node stringToSexpr(const std::string& value)
```

Comment:
What is this for?

Reply:
At first, we wanted to modify the `SmtEngine` API for the two commands to accept sexpr nodes instead of SExpr objects. However, we decided that would be an overkill when the API only allows `std::string`. This function would have translated strings to sexpr nodes and is not necessary anymore. Deleted.